packages/react-app/src/components/field-input/address-field-input.tsx

```diff
@@ -12,7 +12,17 @@ import { IconTooltip } from './icon-tooltip';
 const FieldStyled = styled(Field)`
   ${({ compact }: any) => (compact ? 'margin:0' : '')}
 `;
+const FieldHeaderStyled = styled(Field)`
+  pointer-events: none;
+`;
 
+const LabelStyled = styled.div`
+  width: 16px;
+  justify-content: center;
+  margin-left: 8px;
```

Comment:
GUpx

Reply:
Remove the margin-left (too much margin)
![image](https://user-images.githubusercontent.com/13890618/149633990-62bad9a7-3642-498a-86e0-36e466f82d36.png)

Reply:
Compared to others input